src/ports/favorites/types.ts

```diff
@@ -0,0 +1,23 @@
+export type IFavoritesComponent = {
+  getPicksStatsOfItems(
+    itemIds: string[],
+    userAddress?: string
+  ): Promise<PickStats[]>
+}
+
+export type PickStats = {
+  itemId: string
+  count: number
+  pickedByUser?: boolean
+}
+
+export type FavoritesServerErrorResponse<T> = {
+  ok: boolean
+  message: string
+  data?: T
+}
+
+export type FavoritesServerResponse<T> = {
+  ok: boolean
+  data: T
+}
```

Comment:
If you want, you can use the type we've got in the Marketplace favorite server:
```
export type HTTPResponse<T> = {
  status: StatusCode
  body:
    | {
        ok: false
        message: string
        data?: object
      }
    | {
        ok: true
        data?: PaginatedResponse<T>
      }
    | {
        ok: true
        data?: T
      }
}

export type PaginatedResponse<T> = {
  results: T[]
  total: number
  page: number
  pages: number
  limit: number
}
```
This type will serve as way to type all responses according to our needs.